tgui/packages/tgui/interfaces/PaperSheet.js

```diff
@@ -5,6 +5,7 @@
  * @author Changes stylemistake
  * @author Changes ThePotato97
  * @author Changes Ghommie
+ * @author Changes Timberpoes
```

Comment:
i hate this so much this is just blames but shittier

Reply:
Truer words were never spoken.

Reply:
i hate this as much, this was supposed to be just a copyright header.

Reply:
Copyright headers are for the weak